plugins/encrypted-kubeconfig.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: encrypted-kubeconfig
+spec:
+  version: "v1.0.1"
+  # 'homepage' usually links to the GitHub repository of the plugin
+  homepage: 
+  shortDescription: "Authentication plugin for encrypted KUBECONFIG files."
+  description: |
+    This plugin allows to authenticate to Kubernetes API by decrypting the accesstoken inside of an encrypted KUBECONFIG.
+    This is done using the system keyring.
```

Comment:
Your PR description was also pretty good. I optionally recommend augmenting this field with that.